src/server.ts

```diff
@@ -1,5 +1,10 @@
 import express, { Application, Request, Response } from "express";
 import cors from "cors";
+import * as dotenv from 'dotenv'
+import database from "./config/database";
+import { authRoute } from "./routes/authRoutes";
+import userRouter from './routes/userProfile';
+=======
```

Comment:
What's this line doing here?...Are you sure this is not git conflict?

Reply:
reso;ved conflicts